Multisig/UI/Safe Management/SafeAddressForm/SafeAddressField.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  SafeAddressInput.swift
+//  Multisig
+//
+//  Created by Dmitry Bespalov on 24.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+// needs to be inside a navigation view (eventually) in order for
+// input selector to open the QR scanner and ENS name form on the
+// next screen.
+struct SafeAddressField: View {
+
+    // rendered when text is empty
+    var title: String
+
+    // when not empty, renders in the field according to the "isAddress" logic
+    // and "title" logic
+
+    // when empty, shows the "create safe" prompt under the field
+    var enteredText: Binding<String>
```

Comment:
is it similar to using @Binding or what it the difference?

Reply:
This is to inject the binding. If property wrapper is used, then binding is not injected (or I couldn't find a way how to do it)